src/main/java/com/teachmeskills/employee_info/controllers/InfoController.java

```diff
@@ -0,0 +1,89 @@
+package com.teachmeskills.employee_info.controllers;
+
+import com.teachmeskills.employee_info.model.Employee;
+import com.teachmeskills.employee_info.util.EmployeeInfo;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.Date;
+import java.sql.SQLException;
+
+@Controller
+@RequestMapping("/info")
+public class InfoController {
+    @GetMapping("/get/{id}")
+    public String getInfo(@PathVariable(required = false) Integer id, Model model) {
+        Employee employee = null;
+        try {
+            employee = EmployeeInfo.getInfo(id);
+        } catch (SQLException e) {
```

Comment:
We shouldn't hande SQLException in controller

Reply:
Fixed.

Reply:
What was fixed?)